src/components/NoteItem.tsx

```diff
@@ -9,24 +9,24 @@ interface Props {
 const NoteItem: React.FC<Props> = ({ note, handleOpen }) => {
   return (
     <div key={note.id} className="note-item">
-      <div className="note-item-info">
-        <h3>{note.title}</h3>
-        <p>
+      <div className="note-item-info"         onClick={() => {
+          handleOpen();}}>
+        <h3 className='note-item-title'>{note.title}</h3>
+        {/* <p> */}
           {note.content.split('\n').map((paragraph, index) => (
-            <span key={index}>
+            <div key={index} className='note-item-content'>
```

Comment:
This part is just so newlines `\n` are rendered correctly its not meant to be separate elements.
```suggestion
            <>
```